controllers/server_keys.go

```diff
@@ -44,10 +46,33 @@ type OPRFSeedResponse struct {
 	SeedID int `json:"seedId"`
 }
 
-func NewServerKeysController(opaqueService *services.OpaqueService, jwtService *services.JWTService) *ServerKeysController {
+// TOTPGenerateRequest represents the request body for TOTP key operations
+type TOTPGenerateRequest struct {
+	// AccountID is the UUID of the account for which to generate/delete a TOTP key
+	AccountID uuid.UUID `json:"accountId" validate:"required"`
+	// Email is the email address for the account (used for TOTP generation)
+	Email string `json:"email" validate:"required,email"`
+}
+
+// TOTPGenerateResponse represents the response body containing the generated TOTP key
+type TOTPGenerateResponse struct {
+	// URL is the URL of the TOTP key QR code
+	URL string `json:"url"`
```

Comment:
Technically, this is not a URL since it's not encoding the location of a resource. Maybe we should use URI throughout.